src/kudu/consensus/consensus_queue.h

```diff
@@ -269,6 +269,14 @@ class PeerMessageQueue {
   // Return the next OpId to be appended to the queue in the current term.
   OpId GetNextOpId() const;
 
+  // Get the TrackedPeer corresponding to uuid. The tracked-peer is returned in
+  // 'peer'
+  //
+  // Returns Status::OK() if peer exists.
+  // Returns Status::NotFound() if peer does not exist (and peer is set to
+  // nullptr)
+  Status FindPeer(const std::string& uuid, TrackedPeer* peer);
```

Comment:
since the peer is a return pointer, it should be ** or reference.

Reply:
Intention was to copy the peer into the passed in pointer. Fixed it.